src/Binance/Serialization.cpp

```diff
@@ -19,6 +20,12 @@ static inline std::string addressString(const std::string& bytes) {
     return address.string();
 }
 
+static inline std::string encodeHexBytes(const std::string& bytes) {
```

Comment:
should we move this function into `HexCoding` ? @hewigovens

Reply:
`encodeHexBytes` will generate uppercase string

Reply:
must be upper case?

Reply:
Previously it was. Now we have changed our chain. So lowercase is ok

Reply:
Do we still need this then?

Reply:
Deleted